test/test_subscription.py

```diff
@@ -321,3 +333,61 @@ def test_continuous_subscription(self):
                 self.retrieve(8889, fname, fileVersion=2, targetFile=tmp_path())
         finally:
             subscription_listener.close()
+
+    if ssl is not None:
+        @unittest.skipIf('NGAS_TESTDB' in os.environ, "Need two cluster for this test to run")
```

Comment:
I think this is still preventing the tests from running in Travis because NGAS_TESTDB is always set, but sometimes to an empty value; hence the coverage still doesn't show the code paths being executed. This will need to be changed to `os.environ.get('NGAS_TESTDB', None)` or similar.

Reply:
Also the `if ssl is not None` could be part of the condition within `skipIf` so the whole method won't require an extra level of indentation